### PR TITLE
Add cross-env and fix some Windows issues w scripts

### PR DESCRIPTION
**Pull request by pdehaan**
Doesn't fix the `path.sep` issue, but I'm not seeing that on my Windows 10 cmd.exe config.

Ref #483

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,8 +32,8 @@
     "prepackage": "npm run locales && npm run lint && svgo data/img",
     "prepush": "npm run package",
     "dev": "npm run locales && npm run watch-script & npm run watch & http-server -c-1",
```
Comment (6a68) [inline package.json]: does `http-server ` need a `node` in front of it?

Comment (6a68) [inline package.json]: oh, of course not - it's still on the path, just a windows path.

**Comment by 6a68** [comment]
Thanks @pdehaan! Merging :tada:

File: package.json (hunk 1 above)
Comment (pdehaan) [inline package.json]: As far as I know, `http-server` is a package, not a ./bin/ script, see [package.json:54](https://github.com/meandavejustice/min-vid/blob/327820920aeb1e387e383c8372fe3fd2c62e2157/package.json#L54).

**Pull request merged**